src/openforms/registrations/contrib/objects_api/config.py

```diff
@@ -453,20 +458,40 @@ def _validate_catalogue_and_document_types(attrs: RegistrationOptions) -> None:
         "informatieobjecttype_attachment",
     ):
         url = attrs.get(field)
-        if url:
-            warnings.warn(
-                "URL references to document types are deprecated and will be remove in "
-                "Open Forms 3.0",
-                DeprecationWarning,
-            )
-        if url and url not in informatieobjecttypen_urls:
-            err_tpl = (
-                _("The provided {field} does not exist in the Catalogi API.")
-                if catalogus is None
-                else _("The provided {field} does not exist in the selected catalogue.")
-            )
-            err_msg = err_tpl.format(field=field)
-            raise serializers.ValidationError({field: err_msg}, code="not-found")
+        if not url:
+            continue
+
+        warnings.warn(
+            "URL references to document types are deprecated and will be remove in "
+            "Open Forms 3.0",
+            DeprecationWarning,
+        )
+
+        err_tpl = (
+            _("The provided {field} does not exist in the Catalogi API.")
+            if catalogus is None
+            else _("The provided {field} does not exist in the selected catalogue.")
+        )
+        err_msg = err_tpl.format(field=field)
+
+        if informatieobjecttypen_urls:
```

Comment:
```suggestion
        if informatieobjecttypen_urls is None:
```
it's possible the `catalogus["informatieobjecttypen"]` is equal to `[]` which would cause the code to skip this branch, which would be wrong :)